### PR TITLE
[luci] Extract create_const_node function

### PR DESCRIPTION
**Pull request by jinevening**
This extracts create_const_node function to reduce duplicate codes.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/luci/pass/src/FuseAddWithFullyConnectedPass.test.cpp [hunk 1]
```diff
@@ -27,52 +29,6 @@ namespace
 
 using namespace luci::test;
 
-// TODO Reduce duplicate codes in ResolveCustomOpMatMulPass.cpp
```
Comment (jinevening) [inline compiler/luci/pass/src/FuseAddWithFullyConnectedPass.test.cpp:30]: This PR resolves this TODO.

File: compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp [hunk 2]
```diff
@@ -121,11 +78,12 @@ bool resolve_matmul(luci::CircleCustom *cop)
   if (transpose_a)
   {
     // Create a permutation constant node
-    std::vector<uint32_t> perm;
-    for (uint32_t i = 0; i < circle_lhs->rank(); ++i)
+    std::vector<int32_t> perm;
+    const auto lhs_rank = static_cast<int32_t>(circle_lhs->rank());
+    for (int32_t i = 0; i < lhs_rank; ++i)
```
Comment (jinevening) [inline compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp:83]: This change is necessary to match dtype of "perm" with "perm_node".

File: compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp [hunk 3]
```diff
@@ -141,8 +99,8 @@ bool resolve_matmul(luci::CircleCustom *cop)
   // in row-major order, thus we need to convert between them.
   if (!transpose_b)
   {
-    const std::vector<uint32_t> perm{1, 0};
-    auto perm_node = create_const_node(graph, S32, {2}, perm);
+    const std::vector<int32_t> perm{1, 0};
```
Comment (jinevening) [inline compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp:102]: ditto

File: compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp [hunk 4]
```diff
@@ -121,11 +78,12 @@ bool resolve_matmul(luci::CircleCustom *cop)
   if (transpose_a)
   {
     // Create a permutation constant node
-    std::vector<uint32_t> perm;
-    for (uint32_t i = 0; i < circle_lhs->rank(); ++i)
+    std::vector<int32_t> perm;
```
Comment (seanshpark) [inline compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp:81]: type uint32_t has changed to int32_t. looks like creating S32 requires int32_t. am I correct? but this change seems another reason for this PR. is this change necessary like compiler error?

Comment (seanshpark) [reply compiler/luci/pass/src/ResolveCustomOpMatMulPass.cpp:81]: you've added a comment while I was reading and adding this comment.

**Review by seanshpark — APPROVED** [review]
LGTM
